scripts/docker_deploy_manual.sh

```diff
@@ -19,13 +19,13 @@ cd "$DIR"
 git clone --depth 2 https://github.com/ethereum/solidity.git -b "$branch"
```

Comment:
Yeah, that would be cleaner.

Reply:
> Could you elaborate https://github.com/ethereum/solidity/pull/12717#discussion_r833470661 further ?
> Exactly what change are we looking for ?

@chriseth meant here that instead of building directly inside a checked-out repo maybe we should use `git archive` to build a source archive and build from that instead.

Reply:
github does not allow to use git archive remotely on it's repositories. So, I suppose this option is not available to us

Reply:
You don't have to use it remotely. You can still clone it. The idea is just to unpack the archive to a fresh dir and build in it rather than in a repo checkout.

Anyway, as I said in another thread here, I think it would be fine to just add an issue for that and do it separately to keep the scope of this PR limited.